src/Parser.zig

```diff
@@ -6061,16 +6066,19 @@ fn fieldAccessExtra(p: *Parser, lhs: NodeIndex, record_ty: Type, field_name: Str
                 .ty = f.ty,
                 .data = .{ .member = .{ .lhs = lhs, .index = @intCast(u32, i) } },
             });
-            return p.fieldAccessExtra(inner, f.ty, field_name, false);
+            return p.fieldAccessExtra(inner, f.ty, field_name, false, offset_bits);
```

Comment:
Shouldn't this also take into account the offset of the anonymous struct field? As in:
```c
struct Foo {
    int a;
    struct {
        int b;
    };
};
_Static_assert(__builtin_offsetof(struct Foo, b) == 4, "");

```

Reply:
changed.
I tested this case in a branch w/ layout code. It works now.